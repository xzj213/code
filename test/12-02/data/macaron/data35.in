40
1217 1217 0 0 0 0 1217 1217 1217 0 0 0 0 0 0 1217 1217 0 0 1217 1217 0 1217 1217 1217 0 1217 1217 0 0 0 1217 0 0 0 1217 0 1217 1217 1217
0 1217 1217 0 0 0 0 1217 0 0 1217 0 0 1217 1217 1217 0 1217 1217 1217 1217 0 1217 0 0 0 1217 0 0 0 1217 1217 0 1217 0 0 0 0 0 1217
0 0 0 0 0 0 0 0 0 0 0 1217 0 0 0 1217 0 0 0 1217 1217 1217 0 1217 0 1217 1217 1217 1217 0 1217 0 1217 0 1217 0 0 0 0 1217
1217 0 1217 0 1217 1217 1217 1217 0 1217 0 1217 0 0 1217 0 0 0 1217 1217 1217 0 0 1217 0 0 1217 0 1217 0 0 1217 0 1217 1217 0 1217 0 0 0
1217 1217 1217 1217 0 1217 1217 0 1217 1217 1217 1217 0 1217 0 1217 1217 1217 1217 0 0 0 0 0 0 0 1217 0 0 1217 1217 0 0 0 0 1217 0 1217 0 1217
0 1217 1217 1217 0 0 1217 0 0 1217 1217 1217 1217 1217 0 0 1217 1217 1217 1217 0 1217 1217 1217 1217 0 1217 1217 0 0 1217 1217 1217 1217 0 1217 1217 1217 1217 0
1217 1217 1217 1217 0 1217 0 0 0 1217 0 0 1217 1217 0 0 0 1217 1217 0 0 1217 0 1217 0 0 0 0 1217 1217 1217 0 1217 0 1217 1217 0 1217 0 0
0 1217 1217 0 0 0 1217 1217 0 0 0 0 1217 0 1217 1217 0 1217 1217 0 1217 1217 0 1217 0 0 0 0 0 0 0 1217 0 1217 1217 1217 1217 0 0 1217
0 0 0 0 0 1217 0 0 1217 1217 0 0 0 1217 1217 0 1217 1217 1217 1217 0 0 0 0 1217 1217 1217 1217 1217 1217 1217 0 0 1217 0 0 0 1217 1217 0
0 0 1217 0 0 1217 1217 1217 0 0 0 0 1217 1217 1217 1217 0 0 0 1217 1217 1217 0 1217 0 1217 1217 1217 0 0 1217 1217 1217 0 0 0 1217 0 1217 0
0 1217 1217 1217 1217 1217 0 0 0 1217 1217 0 0 1217 0 0 1217 0 0 0 1217 1217 1217 0 0 0 1217 0 0 0 0 1217 0 1217 0 0 1217 1217 1217 1217
1217 1217 0 1217 0 1217 0 1217 0 0 1217 0 1217 1217 0 1217 1217 0 0 1217 1217 0 0 1217 1217 1217 1217 1217 1217 0 1217 1217 1217 0 1217 1217 0 0 1217 1217
0 0 1217 0 1217 1217 0 0 1217 0 0 0 0 1217 0 1217 1217 1217 1217 0 1217 1217 1217 0 0 1217 0 1217 1217 0 0 1217 0 1217 0 0 1217 1217 1217 0
1217 1217 1217 1217 1217 1217 0 1217 1217 1217 0 1217 1217 0 1217 0 0 0 0 1217 1217 1217 1217 0 0 1217 1217 1217 1217 1217 0 1217 0 1217 0 0 1217 0 1217 0
1217 1217 1217 0 1217 0 1217 1217 1217 1217 1217 1217 0 0 1217 0 1217 1217 0 0 0 1217 0 1217 0 0 1217 1217 1217 0 0 0 0 0 0 1217 1217 1217 1217 1217
1217 0 0 1217 0 1217 1217 0 1217 1217 0 0 0 0 0 1217 0 1217 1217 0 0 0 1217 1217 1217 1217 0 0 0 0 0 1217 0 1217 1217 0 1217 0 0 0
1217 0 1217 0 0 0 1217 0 1217 0 1217 0 0 0 1217 0 0 0 0 0 0 0 0 0 0 0 1217 0 0 0 0 1217 1217 0 0 1217 0 1217 0 1217
0 1217 1217 1217 1217 0 1217 0 1217 1217 1217 1217 1217 0 1217 0 1217 1217 1217 0 1217 0 1217 0 1217 1217 1217 1217 0 0 1217 0 0 1217 1217 1217 1217 1217 1217 1217
0 0 1217 0 0 0 1217 0 1217 1217 1217 0 0 1217 1217 1217 1217 1217 1217 0 0 1217 1217 0 1217 0 0 0 1217 0 0 0 0 0 0 1217 1217 1217 1217 0
1217 0 1217 1217 1217 1217 0 0 0 0 1217 0 1217 0 1217 0 0 0 0 1217 0 0 0 1217 1217 1217 0 1217 0 1217 0 0 1217 0 1217 1217 0 1217 0 1217
0 1217 1217 1217 1217 1217 1217 1217 1217 1217 0 1217 0 1217 0 0 1217 0 1217 0 0 0 0 0 1217 1217 1217 1217 0 0 1217 0 1217 0 1217 1217 1217 1217 1217 1217
1217 1217 1217 1217 0 0 0 0 1217 1217 0 1217 0 0 1217 1217 1217 1217 1217 1217 0 0 0 0 1217 0 0 1217 0 1217 1217 0 1217 1217 1217 1217 0 1217 1217 0
0 0 0 1217 1217 1217 0 1217 1217 1217 1217 0 1217 0 1217 1217 1217 0 0 1217 0 1217 1217 0 0 0 0 0 0 0 1217 0 0 1217 0 1217 1217 1217 1217 0
0 0 1217 0 1217 1217 1217 0 1217 0 1217 1217 1217 0 0 1217 1217 0 1217 1217 0 1217 1217 0 1217 0 1217 0 0 1217 0 0 1217 1217 0 0 0 1217 1217 1217
0 1217 0 1217 0 1217 1217 0 1217 0 0 0 0 0 0 0 1217 1217 0 0 0 1217 0 0 0 0 0 0 1217 1217 0 1217 1217 0 0 0 0 1217 1217 0
0 0 1217 1217 1217 0 0 1217 1217 1217 1217 1217 0 0 0 0 1217 1217 0 1217 0 1217 0 1217 0 1217 1217 1217 0 0 1217 1217 1217 0 1217 1217 1217 0 1217 0
0 0 0 0 0 0 1217 0 0 1217 1217 0 1217 0 0 1217 0 0 1217 1217 1217 0 0 0 0 1217 1217 0 0 0 1217 0 0 0 0 0 0 1217 0 0
0 0 0 0 1217 0 1217 1217 1217 1217 1217 1217 1217 0 1217 1217 1217 0 1217 0 1217 1217 0 1217 1217 1217 1217 0 0 1217 1217 0 1217 1217 1217 1217 1217 0 1217 0
1217 0 0 1217 0 0 0 0 1217 0 1217 0 0 0 1217 1217 0 0 1217 1217 1217 0 0 1217 0 0 1217 1217 1217 0 0 0 1217 0 0 0 0 1217 0 0
1217 0 1217 1217 1217 0 1217 0 1217 0 1217 1217 0 0 1217 0 1217 0 1217 1217 1217 1217 0 1217 1217 1217 1217 1217 0 0 0 0 0 0 1217 1217 1217 0 0 0
0 0 0 1217 0 0 0 0 0 1217 1217 0 1217 1217 0 0 1217 0 0 1217 0 1217 1217 1217 1217 0 0 1217 0 0 1217 0 0 0 0 0 0 1217 0 1217
0 0 0 0 0 0 0 0 0 1217 1217 1217 1217 0 0 0 0 0 1217 0 1217 1217 0 0 0 1217 0 0 1217 0 0 1217 0 0 1217 0 0 1217 0 0
0 0 1217 1217 0 1217 0 0 0 1217 0 1217 0 1217 0 1217 1217 0 1217 0 0 1217 0 1217 1217 1217 1217 1217 1217 1217 0 1217 0 0 1217 0 0 1217 1217 0
0 0 0 1217 1217 0 0 1217 1217 1217 0 0 1217 1217 0 0 1217 1217 1217 1217 0 1217 0 0 0 1217 1217 1217 1217 0 1217 1217 0 0 1217 0 0 1217 1217 0
0 1217 1217 1217 1217 0 1217 1217 0 1217 1217 0 1217 0 0 0 0 1217 0 1217 1832 0 1217 0 0 0 0 1217 1217 1217 0 0 1217 0 0 1217 1217 1217 1217 0
1217 0 1217 1217 1217 0 0 1217 0 1217 0 0 1217 0 1217 1217 1217 1217 0 0 0 1217 1217 2652 1217 0 1217 0 0 1217 1217 0 0 0 1217 0 1217 0 1217 0
1217 1217 1217 0 0 1217 1217 0 0 0 0 1217 0 0 1217 0 0 1217 0 1217 1217 0 0 1217 0 0 0 0 0 0 0 1217 1217 1217 1217 0 0 0 1217 1217
1217 0 0 0 1217 0 1217 1217 1217 1217 0 1217 0 1217 0 0 1217 0 0 1217 0 1217 0 1217 0 1217 0 1217 0 0 1217 1217 0 1217 1217 0 1217 0 0 1217
0 1217 1217 0 0 1217 1217 0 1217 1217 0 1217 1217 0 1217 0 0 1217 1217 0 0 0 0 0 1217 1217 1217 0 1217 0 1217 0 1217 1217 1217 0 1217 0 1217 0
1217 1217 1217 1217 1217 0 0 0 1217 1217 1217 1217 0 1217 1217 0 0 1217 1217 1217 0 1217 1217 0 1217 0 0 0 0 0 1217 1217 0 1217 1217 1217 0 0 1217 1217
