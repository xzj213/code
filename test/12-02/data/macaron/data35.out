0 1001 1217 1832 2485 2652 3444 3741 
